unlock-app/src/components/creator/CreatorLock.js

```diff
@@ -14,7 +14,7 @@ export function CreatorLock({ lock, status = 'deployed' }) {
   let lockComponentStatusBlock
 
   if (status === 'deployed') { // the transaction was mined and confirmed at least 12 times
-    lockComponentStatusBlock = <LockIconBar lock={lock} />
+    lockComponentStatusBlock = <LockIconBarContainer><LockIconBar lock={lock} className={'lock-icons'} /></LockIconBarContainer>
```

Comment:
Pre-emptive: I had to use a container here because you can't reference a real React component directly from a style.

Reply:
Hooo... too bad! LG